.github/ISSUE_TEMPLATE/summit-topic-proposal.md

```diff
@@ -1,14 +1,14 @@
 ---
 name: Summit Topic Proposal
 about: Submit a topic proposal for a session at the Collab Summit
-title: ''
-labels: ''
-assignees: ''
+title: 'Summit Topic: '
+labels: Montreal 2019, Collaborator Summit, Session Proposal
```

Comment:
is this suppose to be 2020?  there is a reference to Montreal 2020 in a change further down

Reply:
good catch, fixed it